src/App.tsx

```diff
@@ -1,103 +1,18 @@
-/* eslint-disable max-len */
 import React from 'react';
-
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
+import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => (
-  <div className="page">
-    <div className="page-content">
```

Comment:
why did you delete the layout of the page?

Reply:
Sorry, seems during moving the code I lost the closing tag and then edited it incorrectly. Didn't notice the sidebar was out of place as the tests ran fine. Perhaps that is why I did not understand what Vasil meant in the comment.
Fixed it now, thanks